### DISCUSSION TITLE
ClassNotFoundException

### DISCUSSION THREAD

**Discussion by ShubhSensei** [quarkusio/quarkus#23448] [Q&A]
## Guys, I'm facing this issue while trying to create a project in VS Code. Can any one please help me how to solve this ?

![Maven wrapper error](https://user-images.githubusercontent.com/91625601/152584034-26b2b9a1-0cad-4974-831b-622ee10599cb.jpg)

**Comment by maxandersen** [comment]
What are you trying to do ? Does not look like a mvn command manually typed in.
